opennms-base-assembly/src/main/filtered/etc/datacollection/mib2-interfaces.xml

```diff
@@ -0,0 +1,38 @@
+<datacollection-group xmlns="http://xmlns.opennms.org/xsd/config/datacollection" name="MIB2-Interfaces">
+   <group name="mib2-interfaces" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.2" instance="ifIndex" alias="ifDescr" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.5" instance="ifIndex" alias="ifSpeed" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.10" instance="ifIndex" alias="ifInOctets" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.11" instance="ifIndex" alias="ifInUcastpkts" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.12" instance="ifIndex" alias="ifInNUcastpkts" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.13" instance="ifIndex" alias="ifInDiscards" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.14" instance="ifIndex" alias="ifInErrors" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.16" instance="ifIndex" alias="ifOutOctets" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.17" instance="ifIndex" alias="ifOutUcastPkts" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.18" instance="ifIndex" alias="ifOutNUcastPkts" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.19" instance="ifIndex" alias="ifOutDiscards" type="counter"/>
+      <mibObj oid=".1.3.6.1.2.1.2.2.1.20" instance="ifIndex" alias="ifOutErrors" type="counter"/>
+   </group>
+   <group name="mib2-X-interfaces" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.1" instance="ifIndex" alias="ifName" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.15" instance="ifIndex" alias="ifHighSpeed" type="string"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.6" instance="ifIndex" alias="ifHCInOctets" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.10" instance="ifIndex" alias="ifHCOutOctets" type="Counter64"/>
+   </group>
+   <group name="mib2-X-interfaces-pkts" ifType="all">
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.7" instance="ifIndex" alias="ifHCInUcastPkts" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.8" instance="ifIndex" alias="ifHCInMulticastPkts" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.9" instance="ifIndex" alias="ifHCInBroadcastPkts" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.11" instance="ifIndex" alias="ifHCOutUcastPkts" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.12" instance="ifIndex" alias="ifHCOutMulticastPkt" type="Counter64"/>
+      <mibObj oid=".1.3.6.1.2.1.31.1.1.1.13" instance="ifIndex" alias="ifHCOutBroadcastPkt" type="Counter64"/>
+   </group>
+   <systemDef name="Enterprise MIB2 Interfaces">
+      <sysoidMask>.1.3.6.1.4.1.</sysoidMask>
+      <collect>
+         <includeGroup>mib2-interfaces</includeGroup>
```

Comment:
I would not include `mib2-interfaces` by default, as we're in 2018, I seriously doubt that it makes sense to duplicate the metrics assuming that there are "slow interfaces" out there.

As a reference, during a consulting with a customer, I saw they were storing 3.3 million metrics every 5 minutes. After some discussion, we were able to validate that all their monitored devices have fast interfaces (i.e. all of them support IF-MIB::ifXTable; in other words, all the interfaces are either FastEthernet or faster), so we disable `mib2-interfaces`, because everything covered by this group is also covered by `mib2-X-interfaces`; which means, it is redundant. In fact, after this simple change, we reduced from 3.3 to 2.1 million metrics to be stored every 5 minutes; and when you're using Newts, this is a tremendous improvement.

My $.02

Reply:
@agalue make sense to me, I'll change it.